### PR TITLE
IDE-2909 Add validation message when there is no files need to be upg…

### PR DESCRIPTION
**Pull request by simonjhy**
…raded

### CODE REVIEW

**Comment by yanan** [comment]
Test Failed on Jenkins http://ide-build-1:9090/ on this commit 6530129fa627c7094438ba9affc5b0847c5c89c0 on build 101

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/AbstractLiferayTableViewCustomPart.java [hunk 1]
```diff
@@ -393,6 +394,12 @@ private IPath createPreviewerFile( final String projectName, final IPath srcFile
                     tableViewElementList.add( tableViewElement );
                 }
             }
+            
+            if ( tableViewElementList.size() == 0 )
+            {
+                final String validationMessage = "No any file need to upgrade.";
+                page.triggerValidationEvent( validationMessage );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/AbstractLiferayTableViewCustomPart.java]: The method `getInitItemsList` should only return the list.  This method should not have the side-effect of changing the UI.  Please redesign and resend, thanks!

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit f9e5a465bcb1406b7951896f1f8d8bf2b3db3d94 on build 107

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit 2f3ddd6a0d98d47cac15ecc3fb191c1e477b7e01 on build 108

**Pull request closed without merging**